extract.py

```diff
@@ -125,12 +125,12 @@ def get_tus(index, path):
             # this works on the assumption that all includes are at the start of the file.
             # calculating the split line gives information about which things are defined in
             # headers and which are defined in in the c file.
-            with open(path + "/" + file, "r") as f:
+            with open(os.path.join(path ,file), "r") as f:
```

Comment:
`(path, file)` (space *after* the comma) conventionally?

Reply:
Thank you for noticing that. Fixed https://github.com/capablevms/function-community-finding/pull/2/commits/7d96f57a92a808e32f5a849e773177a3636e36b1.